ISSUE_TEMPLATE/next.md

```diff
@@ -62,13 +74,18 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 - [ ] Wait for the PR to be approved.
 - [ ] Once approved, merge it and verify that the [`sync-stream-metadata` job](https://jenkins-fedora-coreos.apps.ocp.ci.centos.org/job/sync-stream-metadata/) syncs the contents to S3
 - [ ] Verify the new version shows up on [the download page](https://getfedora.org/en/coreos/download?stream=next)
+    - [ ] x86_64
+    - [ ] aarch64
```

Comment:
depends on https://github.com/coreos/fedora-coreos-tracker/issues/959

Reply:
maybe we should add this later once https://github.com/coreos/fedora-coreos-tracker/issues/959 has been fixed

Reply:
planning on having this fixed in the next week. my lazy self would rather not have to open another PR.

Reply:
on second thought, let's wait to update this when we have a specific link for the arch we can use.. removed.